backbone/src/main/java/org/researchstack/backbone/onboarding/OnboardingSection.java

```diff
@@ -14,11 +15,34 @@
 
 public class OnboardingSection {
 
-    public OnboardingSection() {}
+    /**
+     * These are pre-defined onboarding section identifiers
+     * that are used to create default helper enums for OnboardingSectionType
+     */
+    public static final String LOGIN_IDENTIFIER = "login";
```

Comment:
In the past in iOS, Eric and I have needed to access these identifiers, so I think it' valid to make them public.  

Also, I had to make OnboardingSectionType.LOGIN.getIdentifier() a protected method, because you shouldn't call it.  You should call getTypeIdentifier() on the OnboardinSection object instead. I changed it to this to support unique custom identifiers per enum.